Updates serve modern code to include bugfixes property (#3288)
GoogleChrome/web.dev: src/site/content/en/fast/serve-modern-code-to-modern-browsers/index.md

Ultimately, you should select the appropriate combination of queries to only
target browsers that fit your needs.

### @babel/preset-modules

`@babel/preset-env` groups multiple JavaScript syntax features into collections and enables/disables
them based on the target browsers specified. Although this works well, an entire collection of
syntax features is transformed when a targeted browser contains a bug with just a single feature.
This results in more transformed code than is necessary.

> A separate preset, [`@babel/preset-modules`](https://github.com/babel/preset-modules) solves this

Reviewer: If `bugfixes: true` and `preset-modules` are equivalent, should we even mention the latter? if they are equivalent today, will it remain that way or will the latter be more comprehensive?

Reviewer 2: I took a swing at trying to reword to address this, let me know what you think @housseindjirdeh.

> ### @babel/preset-modules
> 
> `@babel/preset-env` groups multiple JavaScript syntax features into collections and enables/disables
> them based on the target browsers specified. Although this works well, an entire collection of
> syntax features is transformed when a targeted browser contains a bug with just a single feature.
> This results in more transformed code than is necessary.
> 
> A separate preset, [`@babel/preset-modules`](https://github.com/babel/preset-modules) solves this
> problem by converting broken syntax to the closest non-broken, but still modern, syntax that would
> work for the targeted browsers. Instead of needing to install this preset separately, enable the
> [`bugfixes`](https://babeljs.io/docs/en/babel-preset-env#bugfixes) property to add these
> optimizations directly to `@babel/preset-env`.

Reviewer 2: ```suggestion
### Enable Modern Bugfixes

`@babel/preset-env` groups multiple JavaScript syntax features into collections and enables/disables
them based on the target browsers specified. Although this works well, an entire collection of
syntax features is transformed when a targeted browser contains a bug with just a single feature.
This often results in more transformed code than is necessary.

Originally developed as a [separate preset](https://github.com/babel/preset-modules), the 
[bugfixes option](https://babeljs.io/docs/en/babel-preset-env#bugfixes) in `@babel/preset-env`
solves this problem by converting modern syntax that is broken in some browsers to the closest
equivalent syntax that is not broken in those browsers. The result is nearly identical modern code
with a few small syntax tweaks that guarantee compatibility in all target browsers. To use this
optimization, make sure you have `@babel/preset-env` 7.10 or later installed, then set the
[`bugfixes`](https://babeljs.io/docs/en/babel-preset-env#bugfixes) property to `true`:
```

```json
{
 "presets": [
   [
     "@babel/preset-env",
     {
       "bugfixes": true
     }
   ]
 ]
}
```

> This feature is only supported in `@babel/preset-env` version 7.10 or later. Babel 8 will also
> include these optimizations by default.

Reviewer 2: ```suggestion
This feature was introduced in `@babel/preset-env` version 7.10. In Babel 8, the
`bugfixes` option will be enabled by default.
```

Ultimately, you should select the appropriate combination of queries to only
target browsers that fit your needs.

> ### Enable Modern Bugfixes

Reviewer 3: ```suggestion
### Enable modern bugfixes
```

Sentence case is our convention

`@babel/preset-env` groups multiple JavaScript syntax features into collections and enables/disables
them based on the target browsers specified. Although this works well, an entire collection of
syntax features is transformed when a targeted browser contains a bug with just a single feature.
This often results in more transformed code than is necessary.

Originally developed as a [separate preset](https://github.com/babel/preset-modules), the 
[bugfixes option](https://babeljs.io/docs/en/babel-preset-env#bugfixes) in `@babel/preset-env`
solves this problem by converting modern syntax that is broken in some browsers to the closest
equivalent syntax that is not broken in those browsers. The result is nearly identical modern code
with a few small syntax tweaks that guarantee compatibility in all target browsers. To use this
optimization, make sure you have `@babel/preset-env` 7.10 or later installed, then set the
[`bugfixes`](https://babeljs.io/docs/en/babel-preset-env#bugfixes) property to `true`:

```json
{
 "presets": [
   [
     "@babel/preset-env",
     {
       "bugfixes": true
     }
   ]
 ]
}
```

In Babel 8, the `bugfixes` option will be enabled by default.

> ## Use &lt;script type=&quot;module&quot;&gt;

Reviewer 3: ```suggestion
## Use `<script type="module">`
```
Drive-by fix
